public/components/common/modules/discover/row-details.tsx

```diff
@@ -280,8 +281,9 @@ export class RowDetails extends Component {
   // Render the row value column supporting nested fields
   renderArrayValue = (value) => {
     if (arrayContainsObjects(value)) {
-      const formatted = this.props?.indexPattern?.formatHit({ _index: value }, 'html')?._index;
-      return trimAngularSpan(String(formatted));
+      // For compatibility resaons between Kibana 7.10 and 7.16.x we need to normalize the formatHit
```

Comment:
**nitpick**: fix the typo: `resasons`

Reply:
Fixed in https://github.com/wazuh/wazuh-kibana-app/pull/4925/commits/50aa541109011f72ab3ce52208981f9fc8be70e3.